ledger/acctdeltas_test.go

```diff
@@ -716,129 +685,6 @@ func BenchmarkReadingRandomBalancesRAM(b *testing.B) {
 func BenchmarkReadingRandomBalancesDisk(b *testing.B) {
 	benchmarkReadingRandomBalances(b, false)
 }
-func BenchmarkWritingRandomBalancesDisk(b *testing.B) {
```

Comment:
suggestion: this is was a basic benchmark showing sqlite performance. Maybe moving to store/sql_test.go or removing as a separate commit/PR